test/signs/utilities/messages_test.exs

```diff
@@ -13,58 +13,54 @@ defmodule Signs.Utilities.MessagesTest do
   end
 
   defmodule FakePredictions do
-    def for_stop("1", 0) do
-      [
-        %Predictions.Prediction{
-          stop_id: "1",
-          direction_id: 0,
-          route_id: "Red",
-          stopped?: false,
-          stops_away: 1,
-          destination_stop_id: "70093",
-          seconds_until_arrival: 120,
-          seconds_until_departure: 180
-        },
-        %Predictions.Prediction{
-          stop_id: "1",
-          direction_id: 0,
-          route_id: "Red",
-          stopped?: false,
-          stops_away: 1,
-          destination_stop_id: "70093",
-          seconds_until_arrival: 240,
-          seconds_until_departure: 300
-        }
-      ]
-    end
-
-    def for_stop("no_departures", 0) do
-      [
-        %Predictions.Prediction{
-          stop_id: "1",
-          direction_id: 0,
-          route_id: "Red",
-          stopped?: false,
-          stops_away: 1,
-          destination_stop_id: "70093",
-          seconds_until_arrival: 120
-        },
-        %Predictions.Prediction{
-          stop_id: "1",
-          direction_id: 0,
-          route_id: "Red",
-          stopped?: false,
-          stops_away: 1,
-          destination_stop_id: "70093",
-          seconds_until_arrival: 240
-        }
-      ]
-    end
-
     def for_stop(_stop_id, _direction_id), do: []
     def stopped_at?(_stop_id), do: false
   end
 
+  @predictions [
+    %Predictions.Prediction{
+      stop_id: "1",
+      direction_id: 0,
+      route_id: "Red",
+      stopped?: false,
+      stops_away: 1,
+      destination_stop_id: "70093",
+      seconds_until_arrival: 120,
+      seconds_until_departure: 180
+    },
+    %Predictions.Prediction{
+      stop_id: "1",
+      direction_id: 0,
+      route_id: "Red",
+      stopped?: false,
+      stops_away: 1,
+      destination_stop_id: "70093",
+      seconds_until_arrival: 240,
+      seconds_until_departure: 300
+    }
+  ]
+
+  @nd_predictions [
```

Comment:
Is `@nd_predictions` a typo or does `nd` stand for something?

Reply:
It stands for "no departures". I had spelled it out originally, but that caused the auto-formatter to splay out a whole bunch of lines below 😠 .